content/contributing/sprint-guide/contents.lr

```diff
@@ -208,18 +208,21 @@ help - and we want to make sure you have a successful sprint.
 ---
 gutter:
 
+### Help! I'm stuck!
+
 The Bee Team is always available to answer questions. If you're in
 doubt... ask! Don't be afraid to ask *any* question. If you're stuck, we
 want to get you unstuck. If you're confused, it's because the topic is
 complicated, and we haven't explained it well enough. We're eager to
 help - and we want to make sure you have a successful sprint.
 
-**This guide is new!** PyCon US 2025 is the first time we've used it. We
-want your feedback! What worked for you? What didn't? What can we do
-better? Please let us know. And remember: fixing a problem with this
-Sprint Guide counts as a contribution to BeeWare!
+### Improving this guide
+
+If you've got any suggestions on how to improve this sprint guide, let us know.
```

Comment:
Hah! I had this on my todo list for my next PR. I figured it should be separate. Done now!